http://sourceware.org/ml/gdb-cvs/2009-04/msg00204.html
http://sourceware.org/ml/gdb-cvs/2009-04/msg00204.html
d0194c6e1be20c223824e1b1648b4ff0543b1335

### src/gdb/ChangeLog	2009/04/29 07:51:32	1.10423
### src/gdb/ChangeLog	2009/04/29 19:31:55	1.10424
## -1,3 +1,9 @@
+2009-04-29  Pedro Alves  <pedro@codesourcery.com>
+
+	* breakpoint.c (print_exception_catchpoint): Access `b' directly
+	instead of `b->loc->owner'.
+	(print_mention_exception_catchpoint): Ditto.
+
 2009-04-29  Jan Kratochvil  <jan.kratochvil@redhat.com>
 
 	* macrocmd.c (info_macro_command): Print -Dname=value if LINE is zero.
--- src/gdb/breakpoint.c	2009/04/23 22:38:24	1.391
+++ src/gdb/breakpoint.c	2009/04/29 19:31:58	1.392
@@ -6631,7 +6631,7 @@
     breakpoint_adjustment_warning (b->loc->requested_address,
 	                           b->loc->address,
 				   b->number, 1);
-  bp_temp = b->loc->owner->disposition == disp_del;
+  bp_temp = b->disposition == disp_del;
   ui_out_text (uiout, 
 	       bp_temp ? "Temporary catchpoint "
 		       : "Catchpoint ");
@@ -6678,7 +6678,7 @@
   int bp_temp;
   int bp_throw;
 
-  bp_temp = b->loc->owner->disposition == disp_del;
+  bp_temp = b->disposition == disp_del;
   bp_throw = strstr (b->addr_string, "throw") != NULL;
   ui_out_text (uiout, bp_temp ? _("Temporary catchpoint ")
 			      : _("Catchpoint "));
